apps/02RockPaperScissors.js

```diff
@@ -10,19 +10,73 @@ var rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
+  // converting all words to lowercase
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
+
   // Write code here
+  if (hand1 === hand2) {
+    return "It's a tie!";
+  }
+
+  if (hand1 === 'rock') {
+    if (hand2 === 'scissors') {
+      return 'Hand one wins!';
+    }
+    // If we reach here, player 2 must have dealt paper
+    if (hand2 === 'paper') {
+      return 'Hand two wins!';
+    }
+  };
+  // specs for the other combinations
+  if (hand1 === 'paper') {
+    if (hand2 === 'rock')
+      return 'Hand one wins!';
+
+    if (hand2 === 'scissors') {
+      return 'Hand two wins!'
+    }
+  }
+
+  if (hand1 === 'scissors') {
+    if (hand2 === 'paper')
+      return 'Hand one wins!';
+
+    if (hand2 === 'rock') {
+      return 'Hand two wins!'
+    }
+    //setting conditionals for input
+  }
+  if (hand1 !== 'rock', 'paper', 'scissors'); {
+    return 'Not valid entry'
+  }
+  if (hand2 !== 'rock', 'paper', 'scissors');
+  return 'Not valid entry'
 
 }
 
+
+// trying to restart the game
+/*
+function checkForWin() {
+  if ('Hand one wins!', 'Hand two wins!') {
+    return 'Do you want to play again?'
+    if (true) {
+      rockPaperScissors.restart();
```

Comment:
I see what you're trying to do here - you'll learn all about defining methods when we cover Objects.

But in the meantime, [feel free to read up on them](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object) :)